network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/accordion_block.html

```diff
@@ -0,0 +1,20 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block block_content %}
+
+	<div class="tw-row">
+            {% for block in self.accordion_items %}
+                <details class="tw-w-full tw-font-sans tw-font-light tw-mx-4">
+                    <summary class="tw-list-none tw-w-full tw-text-lg tw-leading-7 tw-min-h-[80px] py-3 d-flex align-items-center tw-border-t-gray-20 tw-border-t">
+                        <span class="tw-mr-auto">{{block.title}}</span>
+                        <img src="{% static "_images/plus-circle.svg" %}" alt="" data-state="open" class="summary-open:tw-hidden" />
+                        <img src="{% static "_images/minus-circle.svg" %}" alt="" data-state="close" class="tw-hidden summary-open:tw-block" />
+                    </summary>
+                    {% with item=block count=self.accordion_items|length %}
+                        {{item.content}}
```

Comment:
I believe there will be issues with links in the richtext block. The links are probably not going to be converted from their database from to the external form. To get then into the external form you need to pass richtext content thought he `|richtext` filter.